pangea-sdk/v3/service/vault/integration_test.go

```diff
@@ -50,7 +50,6 @@ func GetName(name string) string {
 }
 
 func GetRandID() string {
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Good one, but we should move Seed to setup function, isn't it?

Reply:
I removed this since `Seed()` is deprecated and we weren't actually utilizing the seed to produce a particular sequence of results (the generator is using a new seed each time, and only to generate a single number). I could adjust things to create a local generator with some constant seed at setup, but since we've come this far without that I feel like it wouldn't be very useful.